tcms/testruns/templates/testruns/get.html

```diff
@@ -117,6 +117,100 @@ <h2 class="card-pf-title" style="text-align: left">
+                                            <ul class="dropdown-menu">
+                                                <li class="selected"><a href="#">{% trans "Name" %}</a></li>
+                                                <li><a href="#">{% trans "Priority" %}</a></li>
+                                                <li><a href="#">{% trans "Category" %}</a></li>
+                                            </ul>
+                                        </div><!-- /input-group-btn -->
+                                        <input type="text" class="form-control" id="filter" placeholder='{% trans "Filter By Name..." %}'>
+                                    </div><!-- /input-group -->
+                                </div>
+                                <div class="form-group">
+                                    <div class="dropdown btn-group">
+                                        <button type="button" class="btn btn-default dropdown-toggle" id="filter-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Name <span class="caret"></span></button>
+                                        <ul class="dropdown-menu">
+                                            <li class="selected"><a href="#">{% trans "Name" %}</a></li>
+                                            <li><a href="#">{% trans "Priority" %}</a></li>
+                                            <li><a href="#">Last {% trans "Category" %}</a></li>
+                                        </ul>
+                                    </div>
+                                    <button class="btn btn-link" type="button">
+                                        <span class="fa fa-sort-alpha-asc"></span>
+                                    </button>
+                                </div>
+                                <div class="form-group">
+                                    <button type="button" class="btn btn-default">{% trans "Add case" %}</button>
+                                    <button type="button" class="btn btn-default">{% trans "Remove case" %}</button>
+                                    <button type="button" class="btn btn-default">{% trans "Update case text" %}</button>
+                                    <button type="button" class="btn btn-default">{% trans "Change assignee" %}</button>
+                                    {% comment %} TODO: dropdown or modal - https://www.patternfly.org/v3/pattern-library/widgets/#modal {% endcomment %}
+                                    <button type="button" class="btn btn-default">{% trans "Change status" %}</button>
+                                    <button type="button" class="btn btn-default">{% trans "Add comment" %}</button>
+                                    <button type="button" class="btn btn-default">{% trans "Add hyperlink" %}</button>
```

Comment:
All of these buttons should bind to their respective actions. It is a good idea to add the actions together with the UI.

Reply:
do you mean to delete the buttons now or to add the actions?

I prefer to merge this with the smallest possible changeset and work on that, than to become a huge PR with a lot of changes in UI, JS, etc.

Reply:
I mean add the actions once the design/placing of the buttons  is done